tempesta_fw/http_msg.c

```diff
@@ -476,6 +476,50 @@ __hdr_del(TfwHttpMsg *hm, int hid)
 }
 
 /**
+ * Optimization that recognizes when duplicate headers immediately follows the
+ * original, and then uses the space of such a headers for the substitution.
+ * Also, it is required that all the headers must reside in one SKB.
```

Comment:
Could you please note that the most expected case for the optimization is Set-Cookie.